### DISCUSSION TITLE
Custom cities

### DISCUSSION THREAD

**Discussion by luischin** [phaselden/FlipIt#28] [Q&A] [answered]
Great little screensaver, thank you. Is there a way one can edit additional cities which are not built-in with time zone info?

**Comment by phaselden** [comment, marked as answer]
Hi. You can add a city to the list that has the same timezone/DST info as the city you want, and then select the new item in the list and Edit its name to be whatever you want it to be.

**Reply by luischin** [reply]
Many thanks!

**Comment by Erich-Chen** [comment]
I want to add up the the full city list can be read in `src/FlipIt/res/TimeZoneCities.txt`.

https://raw.githubusercontent.com/phaselden/FlipIt/master/src/FlipIt/res/TimeZoneCities.txt
